asciidoc-conversion/modules/proc-logging-into-qpc.adoc

```diff
@@ -12,6 +12,6 @@ After the connection between the :product-title_cmd: command line tool and the :
 # :product-title_cmd: server login
 ----
 
-. Enter the server user name and password at the prompts. The default login is admin and the password is '+pass+'.
+. Enter the server user name and password at the prompts. The default login is admin and the password is '+qpcpassw0rd+'.
```

Comment:
change: "default login" to "default user name" 
change "and the password is" to "and the default password is"
change: tagging around "admin" to match tagging around "qpcpassw0rd" (to '+admin+')

Reply:
👍